cpu/stm32/include/periph/f1/periph_cpu.h

```diff
@@ -63,6 +65,48 @@ extern "C" {
  */
 #define ADC_DEVS            (2U)
 
+/**
+ * @name    GPIO Definitions Missing in Vendor Files
+ * @{
+ */
+/**
+ * @brief   Possible values of the MODE0 field in the GPIO CRL register
+ *
+ * The MODE1 to MODE7 fields have the same values. Don't forget to shift the
+ * constants to the field position for MODE1 to MODE7 by 4 times n bits, where
+ * n is the pin number.
+ *
+ * In addition the MODE8 to MODE15 fields in the CRH register have the same
+ * layout and semantics as the MODE0 to MODE 7 fields in the CRL register.
+ */
+enum {
```

Comment:
:+1: We should have use such definitions also in GD32V implementation. So there is some room for cleanups in GD32V implementation :wink: